crates/forge/src/lib.rs

```diff
@@ -54,13 +53,41 @@ static BUILTINS: Lazy<Vec<&str>> = Lazy::new(|| {
     ]
 });
 
+#[derive(Debug, PartialEq)]
+enum TestsToRun {
+    All,
+    Ignored,
+    NonIgnored,
+}
+
+impl TestsToRun {
+    fn from_flags(only_ignored: bool, include_ignored: bool) -> Self {
+        if include_ignored {
+            Self::All
+        } else if only_ignored {
+            Self::Ignored
+        } else {
+            Self::NonIgnored
+        }
+    }
+}
+
+fn should_be_run(tests_to_run: &TestsToRun, case_ignored: bool) -> bool {
+    match tests_to_run {
+        TestsToRun::All => true,
+        TestsToRun::Ignored => case_ignored,
+        TestsToRun::NonIgnored => !case_ignored,
+    }
+}
```

Comment:
What this function does is kind of unclear just looking at its implementation. Perhaps this should be defined on test case?

Reply:
True, will change that

Reply:
`TestCase` is declared in `test-collector`, so it's not possible, but I can pass a test case here as arg

Reply:
That could be better I think